tests/test_env.py

```diff
@@ -429,7 +381,11 @@ def create_install_package(name: str, version: version.Version, requirements: Li
 build-backend = "setuptools.build_meta"
                 """.strip()
             )
-        env.process_env.install_from_source([env.LocalPackagePath(path=str(tmpdir), editable=False)])
+        env.process_env.install_for_config(
+            requirements=[],
+            paths=[env.LocalPackagePath(path=str(tmpdir), editable=False)],
+            config=PipConfig(use_system_config=True),
```

Comment:
Required? I'd think setuptools and wheel would already be installed.

Reply:
seems ok